src/measure_theory/measure/content.lean

```diff
@@ -362,6 +364,56 @@ begin
   exact (μ.le_outer_measure_compacts K).trans (le_to_measure_apply _ _ _)
 end
 
+end outer_measure
+
+section regular_contents
+
+/-- A content `μ` is called regular if for every compact set `K`,
+  `μ(K) = inf {μ(K') : K ⊂ int K' ⊂ K'`. See Paul Halmos (1950), Measure Theory, §54-/
+def content_regular := ∀ ⦃K : topological_space.compacts G⦄,
+  μ K = ⨅ (K' : topological_space.compacts G) (hK: (K : set G) ⊆ interior (K' : set G) ), μ K'
+
+lemma content_regular_exists_compact (H : content_regular μ) (K : topological_space.compacts G)
+  {ε : nnreal} (hε : ε ≠ 0) :
+  ∃ (K' : topological_space.compacts G), (K.carrier ⊆ interior K'.carrier) ∧ μ K' ≤ μ K + ε :=
+begin
+  by_contra hc,
+  simp only [not_exists, not_and, not_le] at hc,
+  have lower_bound_infi : μ K + ε ≤ ⨅ (K' : topological_space.compacts G)
+    (h: (K : set G) ⊆ interior (K' : set G) ), μ K' :=
+    le_infi (λ K', le_infi ( λ K'_hyp, le_of_lt (hc K' K'_hyp))),
+  rw ← H at lower_bound_infi,
+  exact (lt_self_iff_false (μ K)).mp (lt_of_le_of_lt' lower_bound_infi
+    (ennreal.lt_add_right (ne_top_of_lt (μ.lt_top K)) (ennreal.coe_ne_zero.mpr hε))),
+end
+
+variables [measurable_space G] [t2_space G] [borel_space G]
+
+/--If `μ` is a regular content, then the measure induced by `μ` will agree with `μ`
+  on compact sets.-/
+lemma measure_eq_content_of_regular
+ (H : measure_theory.content.content_regular μ) (K : topological_space.compacts G) :
+  μ.measure ↑K = μ K :=
+begin
+  refine le_antisymm _ _,
+  { apply ennreal.le_of_forall_pos_le_add,
+    intros ε εpos content_K_finite,
+    obtain ⟨ K', K'_hyp ⟩ := content_regular_exists_compact μ H K (ne_bot_of_gt εpos),
+    calc μ.measure ↑K ≤ μ.measure (interior ↑K') : _
+                  ... ≤ μ K' : _
+                  ... ≤ μ K + ε : K'_hyp.right,
+
+    { rw [μ.measure_apply ((is_open_interior).measurable_set),
+        μ.measure_apply K.compact.measurable_set],
+      exact μ.outer_measure.mono K'_hyp.left },
+    { rw μ.measure_apply (is_open.measurable_set is_open_interior),
+      exact μ.outer_measure_interior_compacts K' } },
+  { rw (μ.measure_apply (is_compact.measurable_set K.compact)),
```

Comment:
```suggestion
        μ.measure_apply K.is_compact.measurable_set],
      exact μ.outer_measure.mono K'_hyp.left },
    { rw μ.measure_apply (is_open.measurable_set is_open_interior),
      exact μ.outer_measure_interior_compacts K' } },
  { rw (μ.measure_apply (is_compact.measurable_set K.is_compact)),
```
This will fix the errors after merging with master